spec/controllers/lectures_controller_spec.rb

```diff
@@ -2,24 +2,28 @@
 
 RSpec.describe LecturesController, type: :controller do
   let(:valid_attributes) {
-    { name: "SWT", enrollment_key: "swt", is_running: true }
+    { name: "SWT", enrollment_key: "swt", status: "created", questions_enabled: true, polls_enabled: true }
+  }
+  let(:valid_attributes_with_lecturer) {
```

Comment:
Is it not possible to copy the valid_attributes? So if you change one both change? Would be nice tzo have but not must have, so if to time consuming skip this.
- [ ] Check if easy adoptable and if do so

Reply:
Looks complicated to me: https://stackoverflow.com/questions/11853491/merging-two-ruby-objects

Reply:
I think this is not a ruby object (therefore we would need a class) rather it is an Hash Map which can be expanded and new values can be added.

https://stackoverflow.com/questions/6863771/how-to-add-to-an-existing-hash-in-ruby/6864345
https://docs.ruby-lang.org/en/2.0.0/Hash.html
https://gist.github.com/Integralist/9041051#gistcomment-2315237

I think something like

let(:valid_attributes_with_lecturer) {
   {}.merge(:valid_attributes)[:lecturer] = FactoryBot.create(:user, :lecturer, email: "test@test.de")
hash2[:key3] = 'value3'

But maybe this is not so good readable... But I'm also not so firm with let()

Reply:
Hmm I am unable to figure this out.

Reply:
Then skip ;)